docs/resources/service_account_key.md

```diff
@@ -0,0 +1,46 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_service_account_key Resource - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Service Key authorizes access to all Resources assigned to the Service. For more information, see Twingate's documentation https://www.twingate.com/docs/services/#2-select-generate-key-to-create-a-new-service-key.
+---
+
+# twingate_service_account_key (Resource)
+
+Service Key authorizes access to all Resources assigned to the Service. For more information, see Twingate's [documentation](https://www.twingate.com/docs/services/#2-select-generate-key-to-create-a-new-service-key).
+
+## Example Usage
+
+```terraform
+provider "twingate" {
+  api_token = "1234567890abcdef"
+  network   = "mynetwork"
+}
+
+resource "twingate_service_account" "github_actions_prod" {
+  name = "Github Actions PROD"
+}
+
+resource "twingate_service_account_key" "github_key" {
+  name = "Github Actions PROD key"
+  service_id = twingate_service_account.github_actions_prod.id
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `service_id` (String) The id of the service account
```

Comment:
We should also make this parameter `service_account_id` for consistency

Reply:
updated